src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -16,16 +16,19 @@
     get_yaml_template,
     build_timers_info,
     remove_timer_trigger,
-    get_task_id_from_task_name
+    get_task_id_from_task_name,
+    prepare_source_location
 )
 from ._stream_utils import stream_logs
+from ._constants import (
```

Comment:
@ElianaLiberman since we now have a const for `ACR_NULL_CONTEXT`, please also update the reference in run.py